Podcast/SubscriptionsCollectionViewFlowLayout.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  SubscriptionsCollectionViewFlowLayout.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 3/6/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class SubscriptionsCollectionViewFlowLayout: UICollectionViewFlowLayout {
```

Comment:
I believe Kevin implemented his cells using the delegate protocol, it makes it a bit more clear because the view controller is responsible for adjusting the collection view. Unless we plan on using this same flow layout elsewhere, I say make it part of the VC